app/routes/tentang-kami.tsx

```diff
@@ -9,7 +9,7 @@ export const handle: Handle = { name: 'Tentang Kami' }
 
 export const meta: MetaFunction = () => {
   return {
-    title: `Senarai | ${handle}`,
+    title: `Senarai | ${handle.name}`,
```

Comment:
While you're at it, can you please add a new test (either unit test or e2e test) to cover this particular use case by asserting the title tag?
Thanks! 🙏

Reply:
Sure! let me try that